contentcuration/contentcuration/serializers.py

```diff
@@ -194,6 +194,19 @@ class ContentNodeSerializer(BulkSerializerMixin, serializers.ModelSerializer):
     ancestors = serializers.SerializerMethodField('get_node_ancestors')
     files = FileSerializer(many=True, read_only=True)
     associated_presets = serializers.SerializerMethodField('retrieve_associated_presets')
+    metadata = serializers.SerializerMethodField('retrieve_metadata')
+
+    def retrieve_metadata(self, node):
+        all_descendants = node.get_descendants(include_self=True).annotate(max_sort_order=Max('children__sort_order'))
+        resource_descendants = all_descendants.exclude(kind=content_kinds.TOPIC)
+        resource = resource_descendants.exclude(files=None).aggregate(resource_size=Sum('files__file_size'))
```

Comment:
Will probably work OK without `.exclude(files=None)` too -- it'll only aggregate over matching File objects that exist.